spec/jobs/ingest_job_spec.rb

```diff
@@ -128,7 +129,8 @@
     before do
       described_class.perform_now(model_params: model,
                                   background_job_result: result,
-                                  signed_ids: signed_ids,
+                                  signed_ids: {},
+                                  globus_ids: signed_ids,
```

Comment:
It looks like this test is using `signed_ids` for both ActiveStorage file IDs and Globus file IDs. I wonder if it's worth renaming that `let` variable to something like `file_ids`?